src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -174,11 +182,22 @@ def reinit_data(self):
         if not self.loc_port_data:
             logger.warning("0 - b'PORT_TABLE' is empty. No local port information could be retrieved.")
 
+    def _get_if_entry(self, if_name):
+        if_table = ""
+
+        if if_name in self.if_name_map:
+            if_table = mibs.if_entry_table(if_name)
+        elif if_name in self.mgmt_oid_name_map.values():
+            if_table = mibs.mgmt_if_entry_table(if_name)
```

Comment:
>f_table = mibs.mgmt_if_entry_table(if_name) [](start = 13, length = 43)

add 'else' at the end?

Reply:
if_table is "" by default, so the return statement will return None in this case. I think no need to handle 'else'

Reply:
I agree you are right. However, if you add 'else', one less redis IO.

---
In reply to: [220899680](https://github.com/Azure/sonic-snmpagent/pull/88#discussion_r220899680) [](ancestors = 220899680)